cds-snc/cppd-medical-report
# Adding section 3 translations

# Description

Added keys and translations for section 3

https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr

# Test Instructions

1. navigate to */fr/section3
1. ensure that french text is there. 

# Help Requested

- check that the french text matches what's in the french form
